File: challenges/2-tabuada/python/guilhermebleidao/tabuada.py
numero = int(input("Digite um número: "))

for i in range(numero):
    index = i + 1
    print("===========================")
    print(f"A tabuada do número {index} é:" )
    
    for j in range(10):
        multiplicador = j + 1
        multiplicação = index * multiplicador
        print(f"  {index}*{multiplicador} = {multiplicação}")
print("===========================")

File: challenges/2-tabuada/js/ArtToledo/tabuada.js
const prompt = require('prompt-sync')();

let numberChoose = prompt('Digite um numero inteiro qualquer: ');
numberChoose = Number(numberChoose);

for (let numberOfTaboadaToBeMultiplied = 1; numberOfTaboadaToBeMultiplied <= numberChoose; numberOfTaboadaToBeMultiplied++) {
  console.log(`A tabuada do numero ${numberOfTaboadaToBeMultiplied} é:`);
  for (let index = 1; index <= 10; index++) {
    console.log(`${numberOfTaboadaToBeMultiplied} * ${index} = `, numberOfTaboadaToBeMultiplied * index)
  }
  console.log('===========================');
};
